app/components/Feedback/index.tsx

```diff
@@ -27,57 +26,89 @@ type FeedbackProps = {
   pageid: string
   showForm?: boolean
   labels?: boolean
-  upHint?: string
-  downHint?: string
   options?: string[]
-  onSubmit?: (message: string, option?: string) => Promise<any>
+  onSubmit: (message: string, option?: string) => Promise<any>
 }
-const Feedback = ({
-  pageid,
-  showForm,
-  labels,
-  upHint,
-  downHint,
-  options,
-  onSubmit,
-}: FeedbackProps) => {
-  const [showFeedback, setShowFeedback] = useState(false)
+const Feedback = ({pageid, showForm, labels, options, onSubmit}: FeedbackProps) => {
+  const [showThanks, setShowThanks] = useState(false)
   const [showFeedbackForm, setShowFeedbackForm] = useState(false)
+  const [upHover, setUpHover] = useState(false)
+  const [downHover, setDownHover] = useState(false)
+  const [vote, setVote]: any = useState(undefined)
+
+  const thanksRef = useRef<HTMLDivElement | null>(null)
+
+  function sendVote(v: boolean) {
```

Comment:
the `Action` component handles that for you. It already was sending thumb up/down feedback to the backend. At least for articles with pageid - chatbot replies don't, as it's not yet certain where that should go. [This bit here](https://github.com/StampyAI/stampy-ui/pull/638/files#diff-2857f00d7ff227da1cdf159ded8aa76b2082a12a29b5a1a179bc701e575b5b0fL55) is responsible for making sure the backend is properly handled

Reply:
The reason I deleted the action component and kinda went from scratch is because I spent like 40 mins trying to get the styling working so it fit with the design on the figma, but it was just too difficult and I didn't want to mess up other parts of the site which were using the `Action` component. I've now copied the logic for posting to the backend into the `sendVote` function, but would still like for you to verify it.